docs/create_a_project/project_notebook.md

```diff
@@ -1,16 +1,10 @@
----
-layout: default
-nav_exclude: true
----
-
 # Create a project
-{: .no_toc }
```

Comment:
Doing this makes it so that the Notebook appears in the TOC of the page that loads it (create_a_project.md), that's what I wanted to avoid :)